products/wnc-s40hss4+xch.md

```diff
@@ -0,0 +1,108 @@
+---
+layout: page
+title: WNC-S40HSS4+xCh
+banner : images/banner-multichannel-hf.jpg
+thumbnail : images/thumbnail-wnc-s40hss4+xch.png
+excerpt: Subnero's underwater acoustic smart modems with multiple channels for high-speed data acquisition.
+categories: wnc
+section_id: products
+---
+
+## Multi-channel Configuration Modem
+
+Engineered with cutting-edge spatial diversity combining techniques to optimize data extraction across multiple receiving channels, Subnero's multi-channel configuration smart modems redefine underwater communication performance, delivering unparalleled reliability in the most demanding environments. Moreover, their distinctive feature allows for the synchronized recording of raw signals from multiple hydrophones, empowering users with a versatile toolkit for innovative applications, including localization and passive acoustic monitoring.
+
+<div class='pulse-img-div'>
+    <img src="{{site.baseurl}}/images/mc-performance.png" class='pulse-img'>
+</div>
+
+## Applications
+
+- High-speed data transfer for underwater IoT sensors
+- On-demand image transfer
+- Large sensor payload data transfer for marine robotics
+- Underwater networks
+- Rapid environmental assessment.
+- Underwater localization & positioning (e.g. USBL)
+
+## Key Features
+
+- Multiple synchronized receiving channels for enhanced communication performance.
+- 2x communication performance as compared to the [single channel]({{site.baseurl}}/products/wnc-s40hss4.html) equivalents.
+- Ability to choose between 1 - 4 additional receiving channels.
```

Comment:
Would this be 3 additional receiving channels for standard production (as Project Selene is a customisation)?
```suggestion
- Ability to choose between 1 - 3 additional receiving channels.
```